{"uid":"669dcad730a41de8","name":"Valid login","fullName":"Login functionality: Valid login","historyId":"34427650e0f365e1f9051e481bac32df","time":{"start":1759570531389,"stop":1759570540169,"duration":8780},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.208)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff7d23e1eb5+80197]\n\tGetHandleVerifier [0x0x7ff7d23e1f10+80288]\n\t(No symbol) [0x0x7ff7d21602fa]\n\t(No symbol) [0x0x7ff7d215d001]\n\t(No symbol) [0x0x7ff7d214da49]\n\t(No symbol) [0x0x7ff7d214f821]\n\t(No symbol) [0x0x7ff7d214dd66]\n\t(No symbol) [0x0x7ff7d214d7b7]\n\t(No symbol) [0x0x7ff7d214d47b]\n\t(No symbol) [0x0x7ff7d214afec]\n\t(No symbol) [0x0x7ff7d214b87c]\n\t(No symbol) [0x0x7ff7d21643ca]\n\t(No symbol) [0x0x7ff7d22095de]\n\t(No symbol) [0x0x7ff7d21e037a]\n\t(No symbol) [0x0x7ff7d22087fb]\n\t(No symbol) [0x0x7ff7d21e0153]\n\t(No symbol) [0x0x7ff7d21a8b02]\n\t(No symbol) [0x0x7ff7d21a98d3]\n\tGetHandleVerifier [0x0x7ff7d269e83d+2949837]\n\tGetHandleVerifier [0x0x7ff7d2698c6a+2926330]\n\tGetHandleVerifier [0x0x7ff7d26b86c7+3055959]\n\tGetHandleVerifier [0x0x7ff7d23fcfee+191102]\n\tGetHandleVerifier [0x0x7ff7d24050af+224063]\n\tGetHandleVerifier [0x0x7ff7d23eaf64+117236]\n\tGetHandleVerifier [0x0x7ff7d23eb119+117673]\n\tGetHandleVerifier [0x0x7ff7d23d10a8+11064]\n\tBaseThreadInitThunk [0x0x7ffc5de4e8d7+23]\n\tRtlUserThreadStart [0x0x7ffc5e6c8d9c+44]\n\n","statusTrace":"  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 15, in step_impl\n    context.driver.get(\"http://127.0.0.1:8000/login\")\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 483, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.208)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff7d23e1eb5+80197]\n\tGetHandleVerifier [0x0x7ff7d23e1f10+80288]\n\t(No symbol) [0x0x7ff7d21602fa]\n\t(No symbol) [0x0x7ff7d215d001]\n\t(No symbol) [0x0x7ff7d214da49]\n\t(No symbol) [0x0x7ff7d214f821]\n\t(No symbol) [0x0x7ff7d214dd66]\n\t(No symbol) [0x0x7ff7d214d7b7]\n\t(No symbol) [0x0x7ff7d214d47b]\n\t(No symbol) [0x0x7ff7d214afec]\n\t(No symbol) [0x0x7ff7d214b87c]\n\t(No symbol) [0x0x7ff7d21643ca]\n\t(No symbol) [0x0x7ff7d22095de]\n\t(No symbol) [0x0x7ff7d21e037a]\n\t(No symbol) [0x0x7ff7d22087fb]\n\t(No symbol) [0x0x7ff7d21e0153]\n\t(No symbol) [0x0x7ff7d21a8b02]\n\t(No symbol) [0x0x7ff7d21a98d3]\n\tGetHandleVerifier [0x0x7ff7d269e83d+2949837]\n\tGetHandleVerifier [0x0x7ff7d2698c6a+2926330]\n\tGetHandleVerifier [0x0x7ff7d26b86c7+3055959]\n\tGetHandleVerifier [0x0x7ff7d23fcfee+191102]\n\tGetHandleVerifier [0x0x7ff7d24050af+224063]\n\tGetHandleVerifier [0x0x7ff7d23eaf64+117236]\n\tGetHandleVerifier [0x0x7ff7d23eb119+117673]\n\tGetHandleVerifier [0x0x7ff7d23d10a8+11064]\n\tBaseThreadInitThunk [0x0x7ffc5de4e8d7+23]\n\tRtlUserThreadStart [0x0x7ffc5e6c8d9c+44]\n\n","statusTrace":"  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 15, in step_impl\n    context.driver.get(\"http://127.0.0.1:8000/login\")\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 483, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Given the user is on the login page","time":{"start":1759570531390,"stop":1759570540163,"duration":8773},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.208)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff7d23e1eb5+80197]\n\tGetHandleVerifier [0x0x7ff7d23e1f10+80288]\n\t(No symbol) [0x0x7ff7d21602fa]\n\t(No symbol) [0x0x7ff7d215d001]\n\t(No symbol) [0x0x7ff7d214da49]\n\t(No symbol) [0x0x7ff7d214f821]\n\t(No symbol) [0x0x7ff7d214dd66]\n\t(No symbol) [0x0x7ff7d214d7b7]\n\t(No symbol) [0x0x7ff7d214d47b]\n\t(No symbol) [0x0x7ff7d214afec]\n\t(No symbol) [0x0x7ff7d214b87c]\n\t(No symbol) [0x0x7ff7d21643ca]\n\t(No symbol) [0x0x7ff7d22095de]\n\t(No symbol) [0x0x7ff7d21e037a]\n\t(No symbol) [0x0x7ff7d22087fb]\n\t(No symbol) [0x0x7ff7d21e0153]\n\t(No symbol) [0x0x7ff7d21a8b02]\n\t(No symbol) [0x0x7ff7d21a98d3]\n\tGetHandleVerifier [0x0x7ff7d269e83d+2949837]\n\tGetHandleVerifier [0x0x7ff7d2698c6a+2926330]\n\tGetHandleVerifier [0x0x7ff7d26b86c7+3055959]\n\tGetHandleVerifier [0x0x7ff7d23fcfee+191102]\n\tGetHandleVerifier [0x0x7ff7d24050af+224063]\n\tGetHandleVerifier [0x0x7ff7d23eaf64+117236]\n\tGetHandleVerifier [0x0x7ff7d23eb119+117673]\n\tGetHandleVerifier [0x0x7ff7d23d10a8+11064]\n\tBaseThreadInitThunk [0x0x7ffc5de4e8d7+23]\n\tRtlUserThreadStart [0x0x7ffc5e6c8d9c+44]\n\n","statusTrace":"  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 15, in step_impl\n    context.driver.get(\"http://127.0.0.1:8000/login\")\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 483, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\suhas\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":true,"attachmentStep":false,"hasContent":true,"stepsCount":0},{"name":"When the user enters valid username and password","time":{"start":1759570540169,"stop":1759570540169,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"hasContent":false,"stepsCount":0},{"name":"Then the user should be redirected to the dashboard","time":{"start":1759570540169,"stop":1759570540169,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"hasContent":false,"stepsCount":0}],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"hasContent":true,"stepsCount":3},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"login"},{"name":"tag","value":"smoke"},{"name":"feature","value":"Login functionality"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["smoke","login"]},"source":"669dcad730a41de8.json","parameterValues":[]}